back-end/src/main/java/admin/config/WebConfig.java

```diff
@@ -9,12 +10,17 @@
 
 @Configuration
 public class WebConfig implements WebMvcConfigurer {
+
+    @Value("${request.origins}")
+    private String[] origins;
+
     @Override
     public void addCorsMappings(CorsRegistry registry) {
         registry.addMapping("/**")
-                .allowedOrigins("*")
+                .allowedOrigins(origins)
                 .allowedMethods("*")
-                .maxAge(3000);
+                .allowedHeaders("content-type")
+                .exposedHeaders("location");
```

Comment:
```suggestion
                .allowedHeaders(HttpHeaders.CONTENT_TYPE)
                .exposedHeaders(HttpHeaders.LOCATION);
```
이렇게가 어떨까요?

사실 잘 될수도 있는데 둘다 
"Location"
"Content-Type"
으로 알고 있어서요.. 저도 확실하게 아는 건 아니지만 제안해 봅니당!

Reply:
좋은 리뷰 감사합니다.
확인해보니 HttpHeaders.LOCATION 가 결국 "Location"으로 같더라고요! (헤더 이름은 대소 문자를 구분하지 않습니다.)

이전 코드도 잘됨을 테스트 하긴 했는데, 제안대로 HttpHeaders의 Enum을 사용하는게 더 좋을 것 같아서 수정하려고 합니다.
감사합니다.

Reply:
맞아요 대소문자를 구분하지 않는지 몰랐네요 감사합니다~

Reply:
배럴 좋은 피드백 👍

Reply:
오호 정의되어 있는 Enum도 있군요 :+1: